_includes/body-landing.html

```diff
@@ -0,0 +1,880 @@
+  <nav class="navbar header">
+    <div class="container">
+      <div class="navbar-header">
+        <button
+          type="button"
+          class="navbar-toggle collapsed"
+          data-toggle="collapse"
+          data-target="#header-content"
+          aria-expanded="false"
+        >
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="/">
+          <img
+            src="/images/docker-docs-logo.svg"
+            alt="Docker Docs"
+            title="Docker Docs"
+          />
+        </a>
+      </div>
+      <div class="collapse navbar-collapse" id="header-content">
+        <ul class="nav navbar-nav">
+          <li class="active">
+            <a href="/">Home</a>
+          </li>
+          <li>
+            <a href="/develop/">Guides</a>
+          </li>
+          <li>
+            <a href="/engine/">Product Manual</a>
+          </li>
+          <li>
+            <a href="/reference/">Reference</a>
+          </li>
+          <li>
+            <a href="https://www.docker.com/blog/">Articles</a>
+          </li>
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+          <li>
+            {% include theme-switch.html %}
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <section class="hero container">
+    <div class="row justify-content-center">
+      <div class="col-xs-12 text-center">
+        <h2>
+          What can we help you find?
+        </h2>
+      </div>
+    </div>
+    <div class="row justify-content-center">
+      <form
+        class="col-xs-12 col-sm-offset-2 col-sm-8 col-md-offset-3 col-md-6 col-lg-offset-4 col-lg-4"
+        method="GET"
+        action="/search/"
+      >
+        <input
+          class="form-control"
+          name="q"
+          placeholder="Search for guides, terms, commands and more..."
+          type="search"
+          autocomplete="off"
+          spellcheck="false"
+          dir="auto"
+        />
+      </form>
+    </div>
+  </section>
+
+  <section class="container">
+    <div class="row">
+      <div class="col-xs-12 col-sm-6 col-lg-4 card-holder">
+        <a class="card" href="/get-started/" target="_blank">
+          <img src="/images/rocket.svg" alt="get started" />
+          <h5 class="title">Get started</h5>
+          <p>
+            Learn Docker basics and the benefits of containerizing you
+            applications.
+          </p>
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-6 col-lg-4 card-holder">
+        <a class="card" href="/get-docker/">
+          <img
+            src="/images/download-docker.svg"
+            alt="download and install docker"
+          />
+          <h5 class="title">Download and install</h5>
+          <p>
+            Download and install Docker on your machine in a few easy steps.
```

Comment:
@usha-mandya wondering if we considered adding direct links to the install pages for "DD mac", "DD windows", and "Linux" here, to provide a more fluent workflow.

(Perhaps we didn't add them to keep the cards "cleaner", just wondering)

Reply:
Yes, we considered the option to add specific links in each card in the initial mocks, but these were later replaced by the single cards we see now. We can revisit this in phase-2.

Reply:
:+1: SGTM